public/keymaps/e/era_divine_default.json

```
@@ -0,0 +1,23 @@
+{
+  "keyboard": "era/divine",
+  "keymap": "default",
+  "commit": "1112a1071e98a3bd45b519c03edbf89531299322",
+  "layout": "LAYOUT_divine_all",
```

This needs to be a case-sensitive match for [whatever is set in the `info.json` file's `layouts` tree](https://github.com/qmk/qmk_firmware/blob/190738c66d184c865f0bb035c49050c3a1e85787/keyboards/era/divine/info.json#L36).

```suggestion
  "layout": "LAYOUT",
```